src/sagas/deposit.js

```diff
@@ -143,12 +153,32 @@ export function* restoreRedemptionState() {
     REDEMPTION_STEP_MAP[tbtc.Deposit.State.AWAITING_BTC_FUNDING_PROOF] = "/pay"
     REDEMPTION_STEP_MAP[tbtc.Deposit.State.ACTIVE] = "/redemption"
     REDEMPTION_STEP_MAP[tbtc.Deposit.State.AWAITING_WITHDRAWAL_SIGNATURE] = "/redemption/signing"
-    REDEMPTION_STEP_MAP[tbtc.Deposit.State.AWAITING_WITHDRAWAL_PROOF] = "/redemption/confirming"
+    REDEMPTION_STEP_MAP[tbtc.Deposit.State.AWAITING_WITHDRAWAL_PROOF] = "/redemption/prove"
     REDEMPTION_STEP_MAP[tbtc.Deposit.State.REDEEMED] = "/redemption/congratulations"
 
     yield* restoreState(REDEMPTION_STEP_MAP, "redemption")
 }
 
+export function* onStateRestored(depositState) {
+    /** @type {TBTC} */
+    const tbtc = yield TBTCLoaded
+
+    switch(depositState) {
+        case tbtc.Deposit.State.AWAITING_SIGNER_SETUP:
+            yield* getBitcoinAddress()
+            break
+        case tbtc.Deposit.State.AWAITING_BTC_FUNDING_PROOF:
+            yield* autoSubmitDepositProof()
+            break
+        case tbtc.Deposit.State.AWAITING_WITHDRAWAL_SIGNATURE:
+            yield* resumeRedemption()
+            break
```

Comment:
This feels awkward somehow… Not a blocking thing, but more noodling on the whole dealing-with-reloading flow.